source/_integrations/tesla_fleet.markdown

```diff
@@ -40,30 +40,140 @@ The Tesla Fleet API {% term integration %} exposes various sensors from Tesla ve
 
 ## Prerequisites
 
-You must have a [Tesla](https://tesla.com) account, and a [Developer Application](https://developer.tesla.com/en_AU/dashboard).
+You must have:
 
-### Developer Application
-
-You must [create your own application](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-2-create-an-application) for the Tesla Fleet API and configure it as an [application credential](https://my.home-assistant.io/redirect/application_credentials).
-When creating the application, you must set the redirect URL to `https://my.home-assistant.io/redirect/oauth`, but the other URLs can be set as desired. You must also complete both [step 3](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-3-generate-a-public-private-key-pair) and [step 4](https://developer.tesla.com/docs/fleet-api/getting-started/what-is-fleet-api#step-4-call-the-register-endpoint) before the application will be able to make API calls.
+- A [Tesla](https://tesla.com) account
+- A [Developer Application](https://developer.tesla.com/en_US/dashboard)
+- A web domain and host that you can serve your public key file from. Some free options include:
+  - [FleetKey.cc](https://fleetkey.cc)
+  - [AWS S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteHosting.html)
+  - [Cloudflare Pages](https://pages.cloudflare.com/)
```

Comment:
For what it's worth, I was able to use [Cloudflare R2](https://www.cloudflare.com/developer-platform/products/r2/), which was simpler to set up.

Reply:
This is the problem with listing options, there are so many that _could_ be there. I personally just want to push fleetkey.cc since that seems to be really helping people without the technical knowledge to do it themselves.